src/components/layout/ToolbarButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {
+  ForwardRefRenderFunction,
+  PropsWithChildren,
+  forwardRef,
+} from 'react';
+
+import { Button } from '@mui/material';
+
+type Props = {
+  dataCy?: string;
+  onClick: () => void;
+  disabled?: boolean;
+};
+
+const ToolbarButton: ForwardRefRenderFunction<
+  HTMLButtonElement,
+  PropsWithChildren<Props>
+> = (props, ref) => {
+  // structure the custom props from the other ones given by the tooltip
+  const { dataCy, onClick, disabled, ...otherProps } = props;
+  return (
+    <Button
+      ref={ref}
+      {...otherProps}
+      data-cy={dataCy}
+      disabled={disabled}
+      sx={{
+        p: 1,
+        minWidth: '0px',
```

Comment:
Yes this ensures that the buttons stay as small as possible, because the MuiButton has a default `min-width: 64px` rule. 
Default:
<img width="505" alt="Screenshot 2022-07-08 at 12 21 48" src="https://user-images.githubusercontent.com/39373170/177974305-5838a5ed-de24-4a82-bb56-3593a61068b6.png">
With `min-width: 0px`:
<img width="507" alt="Screenshot 2022-07-08 at 12 22 24" src="https://user-images.githubusercontent.com/39373170/177974325-b393bebe-aa67-4f71-9a98-2f687b7556e5.png">